content/social_coding.md

```diff
@@ -138,14 +138,28 @@ When you find a repository with code you would like to reuse, you may look at th
 > ... am I allowed to use it?
 - Contribution guide 
 > ... how to contribute and decision process?
+- Code of Conduct
+> ... how to make clear which behaviors are unacceptable and discouraged? How violations of Code of Conduct will be handled?
 - Trust and community 
 > ... somebody you trust recommended it?
 
 
 ... most of which you have or will learn during this [CodeRefinery](https://coderefinery.org) workshop!
 
+```{challenge} Exercises (15 mn)
+Let's go back to the repository `recipe` we have created at the beginning og this CodeRefinery workshop. This repository contains a guacamole recipe and we would like you to think about what to add in your repository to improve its reusability and encourage external contributions. Using the list given above, pick the items (one or two) you think are the most important. 
+
+If you have not created this repository, you can fork the coderefinery [recipe](https://github.com/coderefinery/recipe).
+
+When online, this exercise is done in breakout rooms where we encourage learners to take 2-3 minutes at the end (before returning to the main room) to exchange on their choices and have a look at your colleagues' repositories. Did you all pick the same items? 
+```
+````{solution} Solution
+- Add a license: it is usually advisable to add a license at the creation of your repository but if you haven't, [https://choosealicense.com/](https://choosealicense.com/) can guide you through the process. Once you have selected a license, it can be added in your repo by entering the link to your repo in the section **Suggest this license**.
```

Comment:
Last sentence, suggestion (?): 
You may add the license by copying the license text into a LICENSE file in your repository or use the **Suggest this license** section on choosealicense.com by entering the link to your repository on the website.

Reply:
Very good suggestion. Much clearer than my sentence! Thanks.